disk/datadog_checks/disk/disk.py

```diff
@@ -65,6 +68,39 @@ def __init__(self, name, init_config, instances):
         self._compile_tag_re()
         self._blkid_label_re = re.compile('LABEL=\"(.*?)\"', re.I)
 
+        self.log.info("entering DISK check")
+    
+        if platform.system() == 'Windows':
+            self._manual_mounts = instance.get('create_mounts', [])
+            if not self._manual_mounts:
+                self.log.info("No manual mounts")
+            else:
+                self.log.info("Manual mounts: {}".format(len(self._manual_mounts)))
+                for manual_mount in self._manual_mounts:
+                    self.log.info("mm: {}".format(manual_mount))
```

Comment:
I would either lower the logging level for these to debug or make them more descriptive (without context it's hard to know what `mm` stands for)